styles/globals.css

```diff
@@ -6,6 +6,7 @@
   --back: 50, 71, 96; /* Theme background color (#324760) */
   --green: 40, 177, 35; /* Success color (#28B123) */
   --terminal-output: #3DCFEF;
+  --incorrect-answer: #EF960B;
```

Comment:
Yay thanks for adding this! @GBKS if you have any opinions on the color to use for incorrect answers (at least for the demo), speak now or forever hold your peace (until v1)

Or @sabakhilji! Are there any good accessibility choices here? Right now we're using orange, a color I arbitrarily picked because red felt a little too scary, harsh, and punishing.

Reply:
Love the idea of keeping correct things in white color and I love this orange color. My only concern is about the contrast ratio which might create an accessibility issue. Will dig more into the color scheme to make it more accessible.

Reply:
We can change it to whatever we decide is best, I think a combination of accessibility and user feedback is the way to go here

Reply:
If we're looking into accessibility and contrast, let's please use WCAG 3 standards and not the outdated WCAG 2.1. Most color contrast checking tools use an old model from 2008 based on simple color value math that does not take into consideration how humans perceive color or that screens and accessibility features have improved since then. Thanks.

Reply:
Everyone okay with moving forward with this orange then following up with a later PR once more research is done?

Reply:
Yep.